src/Kiota.Builder/Writers/Php/CodeMethodWriter.cs

```diff
@@ -241,10 +233,10 @@ private void WriteSerializerBody(CodeMethod codeMethod, CodeClass parentClass, L
                 writer.WriteLine($"parent::serialize({writerParameterName});");
             var customProperties = parentClass.GetPropertiesOfKind(CodePropertyKind.Custom).Where(static x => !x.ExistsInBaseType && !x.ReadOnly);
             foreach(var otherProp in customProperties) {
-                writer.WriteLine($"{writerParameterName}->{GetSerializationMethodName(otherProp.Type)}('{otherProp.SerializationName ?? otherProp.Name.ToFirstCharacterLowerCase()}', $this->{otherProp.Name.ToFirstCharacterLowerCase()});");
+                writer.WriteLine($"{writerParameterName}->{GetSerializationMethodName(otherProp.Type)}('{otherProp.SerializationName ?? otherProp.Name.ToFirstCharacterLowerCase()}', $this->get{otherProp.Name.ToFirstCharacterUpperCase()}());");
```

Comment:
would it be possible instead to navigate to the getter from the property to avoid duplicating the naming convnetion?